tools/llm_bench/llm_bench_utils/pt_utils.py

```diff
@@ -131,6 +131,7 @@ def create_image_gen_model(model_path, device, **kwargs):
             model_class = PT_MODEL_CLASSES_MAPPING[model_type]
             start = time.perf_counter()
             pipe = model_class.from_pretrained(model_path)
+            pipe = set_bf16(pipe, device, **kwargs)
```

Comment:
Is set_bf16 required or can it be made optional?

Reply:
It is based on the conditions inside the `set_bf16` funciton, if it is passed in load_config flag, it sets the model to bf16 otherwise skips it.